libs/openFrameworks/3d/ofEasyCam.h

```diff
@@ -163,52 +160,70 @@ class ofEasyCam : public ofCamera {
 	/// Returns the area bounds used for mouse control.
 	ofRectangle getControlArea() const;
 	
+	
+	enum easyCamTransformType{
```

Comment:
Can we Change the name to `EasyCamTransformType` (to fix caps) or just `TransformType`, since it is already in the `ofEasyCam` class?